save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -19,86 +21,144 @@ import react.dom.html.ReactHTML.option
 import react.dom.html.ReactHTML.select
 import react.useState
 
+val testExecutionFiltersRow = testExecutionFiltersRow()
+
 @Suppress("MISSING_KDOC_TOP_LEVEL", "UtilityClassWithPublicConstructor")
 class SelectOption {
     companion object {
         const val ANY = "ANY"
     }
 }
 
+/**
+ * [Props] for filters value
+ */
+external interface FiltersRowProps : Props {
+    /**
+     * All filters in one class property [filters]
+     */
+    var filters: Filters
+
+    /**
+     * lambda to change [filters]
+     */
+    var onChangeFilters: (Filters) -> Unit
+}
+
 /**
  * A row of filter selectors for table with `TestExecutionDto`s. Currently filters are "status" and "test suite".
  *
- * @param initialValueStatus initial value of `TestResultStatus`
- * @param initialValueTestSuite initial value of `test suite`
- * @param onChangeStatus handler for selected value change
- * @param onChangeTestSuite handler for input value
  * @return a function component
  */
-@Suppress("TOO_LONG_FUNCTION", "LongMethod")
-fun testExecutionFiltersRow(
-    initialValueStatus: String,
-    initialValueTestSuite: String,
-    onChangeStatus: (String) -> Unit,
-    onChangeTestSuite: (String) -> Unit,
-) = FC<Props> {
-    val (testSuite, setTestSuite) = useState(initialValueTestSuite)
-    val (status, setStatus) = useState(initialValueStatus)
+@Suppress("TOO_LONG_FUNCTION", "TOO_MANY_PARAMETERS", "LongMethod")
```

Comment:
suppress of `TOO_MANY_PARAMETERS` seems useless here

Reply:
Done